trunk/includes/pardot-settings-class.php

```diff
@@ -9,36 +10,40 @@
  * @author Mike Schinkel <mike@newclarity.net>
  * @since 1.0.0
  */
-class Pardot_Settings {
+class Pardot_Settings
+{
+
+	const BASE_PARDOT_DOMAIN = 'pardot.com';
```

Comment:
Now, if we need to change the Pardot environment used for testing, we just need to update this one constant.